sagan/train.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
```

Comment:
`train.py` should be outside the `sagan` module

Reply:
Cool